recipes/chainmap/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set name = "chainmap" %}
+{% set version = "1.0.1" %}
+{% set sha256 = "c1cd76c679dd7af982ec5a45788ef4a0dfc20e77f27ccdeca289a5141862ff64" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  requires:
+    - pytest
+  imports:
+    - chainmap
+  files:
+    - test_chainmap.py
+  commands:
+    - pytest --pyargs test_chainmap.py
+
+about:
+  home: https://bitbucket.org/jeunice/chainmap/overview
+  license: PSFL
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE.txt'
```

The recipe directory is checked by default if it isn't found in the work directory so this can be simplified:
```suggestion
  license_file: LICENSE.txt
```